GeoLib/Grid.h

```diff
@@ -201,6 +201,10 @@ Grid<POINT>::Grid(InputIterator first, InputIterator last,
                                     _max_pnt[1] - _min_pnt[1],
                                     _max_pnt[2] - _min_pnt[2]}};
 
+    // enlarge delta
+    for (auto & d : delta)
+        d = std::nextafter(d, std::numeric_limits<double>::max());
```

Comment:
I'm not really happy with this. I think the result of `std::nextafter` can be cancelled out by very many arithmetic operations (e.g. adding or subtracting a large number). I think multiplying with `1+eps` would be more robust, but I don't know if that has any implications for our current code.
So if nobody else objects and the solution works for you: :fast_forward: